### ISSUE TITLE
Builds broken on arm7 (32-bit arch)

### ISSUE THREAD

**Issue by dhui** [golang-migrate/migrate#420]
**Describe the Bug**
Releases do not contain pre-built binaries due to an [error](https://travis-ci.com/github/golang-migrate/migrate/jobs/365252615):
```
cd ./cmd/migrate && CGO_ENABLED=0 GOOS=linux GOARCH=arm GOARM=7 go build -a -o ../../cli/build/migrate.linux-armv7 -ldflags='-X main.Version=4.12.0 -extldflags "-static"' -tags 'postgres mysql redshift cassandra spanner cockroachdb clickhouse mongodb sqlserver firebird neo4j file go_bindata github github_ee aws_s3 google_cloud_storage godoc_vfs gitlab' .

# github.com/neo4j/neo4j-go-driver/neo4j/internal/packstream

../../../../../../pkg/mod/github.com/neo4j/neo4j-go-driver@v1.8.0/neo4j/internal/packstream/packer.go:130:10: constant 4294967295 overflows int

../../../../../../pkg/mod/github.com/neo4j/neo4j-go-driver@v1.8.0/neo4j/internal/packstream/packer.go:167:9: constant 4294967296 overflows int

Makefile:11: recipe for target 'build-cli' failed
```

This appears to be caused by the new neo4j Go driver.

Easiest and quickest way to fix would be to drop support/builds for arm7 or neo4j.

**Steps to Reproduce**
Run: `make build-cli` **or** `GOOS=linux GOARCH=arm GOARM=7 go build -tags 'postgres mysql redshift cassandra spanner cockroachdb clickhouse mongodb sqlserver firebird neo4j file go_bindata github github_ee aws_s3 google_cloud_storage godoc_vfs gitlab' .`

**Expected Behavior**
Builds should succeed

**Migrate Version**
v4.12.0

**Comment by dhui** [comment]
Opened a [PR](https://github.com/neo4j/neo4j-go-driver/pull/125) to fix the upstream neo4j driver

**Comment by dhui** [comment]
Not sure if more people use arm7 or neo4j e.g. how to minimize impact of a quick fix

**Comment by 2hdddg** [comment]
Pull request for this is now here: https://github.com/neo4j/neo4j-go-driver/pull/129

**Comment by dhui** [comment]
Fixed in [v4.12.2](https://github.com/golang-migrate/migrate/releases/tag/v4.12.2)
